src/sys/mman.rs

```diff
@@ -48,7 +48,7 @@ libc_bitflags!{
         /// Put the mapping into the first 2GB of the process address space.
         #[cfg(any(all(any(target_os = "android", target_os = "linux"),
                       any(target_arch = "x86", target_arch = "x86_64")),
-                  all(target_os = "linux", target_env = "musl", any(target_arch = "x86", target_pointer_width = "64")),
+                  all(target_os = "linux", target_env = "musl", any(target_arch = "x86", target_arch = "x86_64")),
```

Comment:
Emm, isn't this branch 

`all(target_os = "linux", target_env = "musl", any(target_arch = "x86", target_arch = "x86_64"))` 

already covered by 

`all(any(target_os = "android", target_os = "linux"), any(target_arch = "x86", target_arch = "x86_64"))`